docs/httpproxy.md

```diff
@@ -400,12 +382,45 @@ spec:
           port: 80
 ```
 
+In the following example, we match on headers and send to different services, with a default route if those do not match.
+
+```yaml
+# httpproxy-multiple-headers.yaml
+apiVersion: projectcontour.io/v1alpha1
+kind: HTTPProxy
+metadata:
+  name: multiple-paths
+  namespace: default
+spec:
+  virtualhost:
+    fqdn: multi-path.bar.com
+  routes:
+    - conditions:
+        header:
```

Comment:
Conditions are an array so need to update yaml here. I can fix in a follow-up PR to keep this doc moving.